reference/toolstacks/php/drupal8/developing-with-drupal.md

```diff
@@ -0,0 +1,113 @@
+# Developing with Drupal
+
+## Push changes to an environment
+
+Here, we'll see how to make code changes to an environment.
+
+> **note**
+> You should never be working on the Master branch since it's supposed to be your production environment.
+
+Make sure you're on a working environment. In this example we're on the
+*sprint1* branch:
+
+```bash
+$ git branch
```

Comment:
Hm. The current example is just checking what branch we're on, rather than changing to it.  It's in the current D7 docs and I didn't change it.  You're right, we probably should, though.  I will do so.